sparkle/platform/output.py

```diff
@@ -0,0 +1,298 @@
+        solver_dir_name = path.name
+        scenario_dir = path / "outdir_train_configuration" / \
+            f"{solver_dir_name}_scenario" / "state-run0"
+        if scenario_dir.is_dir():
+            scenario_file = scenario_dir / "scenario.txt"
+            if not scenario_file.is_file():
+                print("Can't find scenario file")
+                return
+        else:
+            print("Can't find scenario file")
+            return
+
+        # Sets scenario on configurator object
+        self.initalise_scenario(scenario_file)
+        # Retrieves validation results and best configuration
+        self.training, self.best_config =\
+            self.get_validation_data(self.configurator.scenario.instance_set)
+        # Retrieve test validation results if they exist
+        if self.instance_set_test is not None:
+            self.test, _ = self.get_validation_data(self.instance_set_test)
+
+        self.write_output()
+
+    def initalise_scenario(self: ConfigurationOutput, scenario_file: Path) -> None:
+        """Reads scenario file and initalises ConfigurationScenario."""
+        config = {}
+        with scenario_file.open() as file:
+            for line in file:
+                key, value = line.strip().split(" = ")
+                config[key] = value
+
+        instance_set_train = resolve_object_name(
+            config["instance_file"],
+            gv.file_storage_data_mapping[gv.instances_nickname_path],
+            gv.instance_dir, InstanceSet)
+        # TODO: Find out why _train.txt is added to end of name
+        # Remove .txt to not cause any issues with later use of
+        # name in self.configurator.scenario._set_paths()
+        instance_set_train.name = instance_set_train.name.replace("_train.txt", "")
+
+        # Set instance_set_test to later retrieve validation results
+        instance_set_test = resolve_object_name(
+            config["test_instance_file"],
+            gv.file_storage_data_mapping[gv.instances_nickname_path],
+            gv.instance_dir, InstanceSet)
+        if instance_set_test is not None:
+            self.instance_set_test = instance_set_test
+
+        # Collect relevant settings
+        if "cpu_time" in config:
+            cpu_time = int(config["cpu_time"])
+        else:
+            cpu_time = None
+
+        if "wallclock-limit" in config:
+            wallclock_limit = int(config["wallclock-limit"])
+        else:
+            wallclock_limit = None
+
+        if "runcount-limit" in config:
+            solver_calls = int(config["runcount-limit"])
+        else:
+            solver_calls = None
+
+        if "feature_file" in config:
+            use_features = bool(config["feature_file"])
+        else:
+            use_features = None
+
+        # TODO: Add METRIC to objective
+        objective = SparkleObjective(f"{config['run_obj']}:UNKNOWN")
+
+        # TODO: Number_of_runs isn't part of the scenario file
```

Comment:
Perhaps we can do some counting in the output directory? Like how many files are in some output dir. Are the number_of_runs is the one from the settings [configuration] section, then I think the number of Slurm output logs is equal to this amount. Hacky, but if it works...